lib/_http_outgoing.js

```diff
@@ -408,7 +407,7 @@ OutgoingMessage.prototype.write = function(chunk, encoding, callback) {
     return true;
   }
 
-  if (!util.isString(chunk) && !util.isBuffer(chunk)) {
+  if (!(typeof chunk === 'string' || util.isBuffer(chunk))) {
```

Comment:
Ah, didn't realize that. I think there's a few other places I could pop that in, thanks.